docs/src/main/paradox/java/http/client-side/pool-overflow.md

```diff
@@ -1,53 +0,0 @@
-# Pool overflow and the max-open-requests setting
-
-@ref[Request-Level Client-Side API](request-level.md) and @ref[Host-Level Client-Side API](host-level.md)
-use a connection pool underneath. The connection pool will open a limited number of concurrent connections to one host
-(see the `akka.http.host-connection-pool.max-connections` setting). This will limit the rate of requests a pool
-to a single host can handle.
-
-When you use the @ref[stream-based host-level API](host-level.md#using-the-host-level-api-in-a-streaming-fashion)
-stream semantics prevent that the pool is overloaded with requests. On the other side, when a new request is pushed either using
-`Http.singleRequest()` or when materializing too many streams using the same `Http().cachedHostConnectionPool`, requests
```

Comment:
Sure, I usually did a quick read after merge, but this one completely went under my radar.